package.json

```diff
@@ -3,15 +3,15 @@
   "version": "0.0.0",
   "private": true,
   "hexo": {
-    "version": "3.3.7"
+    "version": "3.3.8"
   },
   "scripts": {
     "build": "hexo generate && gulp",
     "deploy": "hexo deploy"
   },
   "dependencies": {
     "cheerio": "^0.20.0",
-    "hexo": "hexojs/hexo",
+    "hexo": "github:hexojs/hexo",
```

Comment:
This file should be keep intact since i18n should not have impact on package information.